### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by jume-dev** [profclems/glab#754] [General]
Hi,
sorry to bother you again with this but could you give an estimation on when the new release will come out?

**Comment by profclems** [comment]
I had wanted to do a release together with hosting our new docs site because we generate the docs automatically at every new release. 
I sent a request to Fosshost for a custom domain name (glab-cli.io) and they are willing to help us with a domain name but it has delayed.
I will go ahead and do the release tonight regardless.
Thanks!

**Comment by profclems** [comment]
Finally released: https://github.com/profclems/glab/releases/v1.18.0

**Reply by jume-dev** [reply]
awesome, thank you
